### ISSUE TITLE
No feedback of the screen reader navigating between voices

### ISSUE THREAD

**Issue by Carlos-EstebanM** [musescore/MuseScore#20000]
### Issue type

Other type of issue

### Bug description

Accessibility bug: If a blind person uses Muse Score with a screen reader, and he needs to work with voices, the screen readers does not announce nothing when the commands control + ALT + numbers are pressed. For know if a note or a chord was write in a different voice, is necessary to navigate in the score manually.

### Steps to reproduce

1. Turn on a screen reader, and then, open a file in MuseScore. For test purposes, open a score with only one voice.
2. Go to a beat, and press n to introduce notes.
3. Write a note in the voice one.
4. Press control + ALT + 2. The screen reader does not announce that now you are in the voice two.
5. Write a note.
6. Go back to the voice one (press control + ALT + 1). The screen reader is not announcing the change, again.
7. Press escape. Navigate with right and left arrow keys, and ALT + right and left arrow keys. When you found a note in a different voice, the screen reader say that the note is in the voice one, or in the voice two.

### Screenshots/Screen recordings

https://1drv.ms/u/s!AuzdCVDdF5S-lKcSwXb2AOobtuOpiw?e=2z4civ

### MuseScore Version

4.1.1.232071203

### Regression

I don't know

### Operating system

Windows 11 23h2

### Additional context

Tested with NVDA 2023.3 and Narrator.
